src/qt/overviewpage.h

```diff
@@ -50,16 +49,10 @@ public Q_SLOTS:
     Ui::OverviewPage *ui;
     ClientModel *clientModel;
     WalletModel *walletModel;
-    CAmount currentBalance;
-    CAmount currentUnconfirmedBalance;
-    CAmount currentImmatureBalance;
-    CAmount currentAnonymizedBalance;
-    CAmount currentWatchOnlyBalance;
-    CAmount currentWatchUnconfBalance;
-    CAmount currentWatchImmatureBalance;
     int nDisplayUnit;
     bool fShowAdvancedPSUI;
     int cachedNumISLocks;
+    interface::WalletBalances m_balances;
```

Comment:
this should be under `WalletModel *walletModel;` imo

Reply:
YOU want to break up with upstream? :D

Reply:
No, that would be most consistent with upstream imo

Reply:
ahh got you wrong initially :D not sure why but i thought you want to move the `m_balances` stuff into `WalletModel`